src/qat/purr/integrations/qasm.py

```diff
@@ -816,10 +816,12 @@ def _walk_program(self, prog: ast.Program, context: QasmContext):
             self.visit(node, context)
 
     def visit_Include(self, node: ast.Include, context: QasmContext):
-        if node.filename in ("stdgates.inc", "qelib1.inc"):
+        if node.filename == "qelib1.inc":
             file_path = Path(
                 find_spec("qiskit.qasm.libs").submodule_search_locations[0], node.filename
             )
+        elif node.filename == "stdgates.inc":
```

Comment:
can't we just reuse `find_spec` to locate the same file ?
oh or is the `"stdgates.inc"` file not available in the qiskit release we depend on ?